libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -50,6 +50,14 @@ describe('AccordionItemComponent', () => {
         visibility: 'visible',
       });
     });
+    it('should emit the toggle event when expanded', () => {
+      spyOn(spectator.component.toggle, 'emit');
+      spectator.component.isDisabled = false;
+      spectator.component.isExpanded = false;
+      spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
+      expect(spectator.component.toggle.emit).toHaveBeenCalledTimes(1);
+      expect(spectator.component.toggle.emit).toHaveBeenCalledWith(true);
```

Comment:
Consider readability - separate code blocks in [arrange-act-assert](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80):
```suggestion
      spyOn(spectator.component.toggle, 'emit');
      spectator.component.isDisabled = false;
      spectator.component.isExpanded = false;
      
      spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
      
      expect(spectator.component.toggle.emit).toHaveBeenCalledTimes(1);
      expect(spectator.component.toggle.emit).toHaveBeenCalledWith(true);
```